### DISCUSSION TITLE
希望大世界新增功能

### DISCUSSION THREAD

**Discussion by jn930** [LmeSzinc/AzurLaneAutoScript#2224] [Ideas]
1.大世界第一隊無法戰鬥後自動換到第二隊，以此類推
2.大世界可以自動使用背包中的回復藥

**Comment by ec7529a2** [comment]
1. 这个功能需要有开关，因为部分玩家的队伍有明确分工，用boss队打道中肯定不行，另外，alas可以自动回港修船。
2. 这个功能已经在issue区有feature request了（#867），但是开发者觉得收益比较低，所以还没做。
